scripts/generate_error_code.hs

```diff
@@ -87,6 +87,9 @@ invalidInputErrors = errorsEnumerate 100
   , "position_liquidity_below_zero_err"
       :? "Liquidity of a position went below zero."
 
+  , "incorrect_tick_spacing_err"
+      :? "Tick index is not multiple of tick spacing."
```

Comment:
Minor nitpick, I would change this to:
```suggestion
      :? "Tick indexes must be a multiple of the tick spacing."
```
to be in the same format the other errors.

Reply:
Updated this.